net/net.sh

```diff
@@ -175,6 +175,7 @@ startBootstrapNode() {
       rsync -vPrc -e "ssh ${sshOptions[*]}" "$SOLANA_ROOT"/solana-release/bin/* "$ipAddress:~/.cargo/bin/"
       ;;
     local)
+      rsync -vPrc -e "ssh ${sshOptions[*]}" "$SOLANA_ROOT"/farf/*.so "$ipAddress:~/.cargo/bin/"
```

Comment:
I think @mvines is already taking care of this issue in the PRs linked to this issue: https://github.com/solana-labs/solana/issues/2065

Reply:
will revert this change

Reply:
Yep, if #2069 doesn't work for you please let me know.  I'll land it as soon as CI is green for it